fetch-config.sh

```diff
@@ -2,10 +2,33 @@
 
 # Downloads DEV config to /etc/gu
 
+set -e
+
+# colours
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+NC='\033[0m' # no colour - reset console colour
+
 DOWNLOAD_DIR=/etc/gu
 
-aws s3 cp s3://grid-conf/DEV/ \
+downloadConfig() {
+  aws s3 cp s3://grid-conf/DEV/ \
     ${DOWNLOAD_DIR}/ \
     --recursive \
     --profile media-service \
     --region eu-west-1
+}
+
+if [[ -w ${DOWNLOAD_DIR} ]]; then
+  downloadConfig
+  echo -e "${GREEN}Done ${NC}"
+else
+  echo "Cannot write to ${DOWNLOAD_DIR}. It either doesn't exist or is not owned by $(whoami)."
+
+  echo "Creating ${DOWNLOAD_DIR} and making user $(whoami) the owner"
```

Comment:
Maybe worth adding a message that this will prompt you for your sudo password

Reply:
Done.